src/dtproject/assets/schemas/configurations.yaml.json

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Duckietown Dev Container Configurations",
+    "type": "object",
+    "patternProperties": {
+      ".*": {
+        "type": "object",
+        "properties": {
+          "configuration": { "type": "string", "default": "default" },
+          "devcontainer": { "type": "string", "default": "devcontainers" },
```

Comment:
There is no pointer from `containers.yaml` to `devcontainers.yaml`, only the other way around.

Reply:
Removed these two properties.